CHANGES.rst

```diff
@@ -3,6 +3,9 @@
 
 * Fix #130 pedantic table parsing not throwing exception.
 
+* PR #143 gets the docs build working again and required to pass CI.
```

Comment:
it's up to you to decide the policy, but normally we keep the approach of only adding stuff into the changelog for user facing changes, e.g. no entry for docs, test, internal infrastructure.

Reply:
Okay I put it in for the build break (also not user facing), so I figured it's probably better to have it in here than not.  Maybe there should be two sections, one with the features, and then everything else?

I thought a while ago you were suggesting the opposite, how every PR should have a changes.rst entry

Reply:
no, I never suggested that every, but suggested to enable the bot that keeps track that stuff that needs the entry gets it, and that it's consistent (it assumes there is a `no-changelog-entry-needed` label for stuff that doesn't need the entry).

Reply:
OK, well that seems a bit heavy for me, so I think I'll just make two sections, one for misc fixes (which always end up seeming to have more impact than we think), and one for general user features, and we'll just keep sure that major things are in the user features then.

Reply:
labels are useful anyway

Reply:
I too think that this file is a one stop for changes in software such as: bug fixes, new features, deprecated functionality, dependency updates - we could even decide to organize it along those lines. Users are not interested in the other changes. Those will be for us and I think GH does a good job of keeping track of those if we need them.
As for the label, when it fails, the bot could remind us about its existence.

Reply:
Okay I talked with @andamian and he helpfully agreed to take on setting up the bot.  I'm happy to go with that and figure out these labels once we get the bot going, and take this out of the changelog